source/logrepl/snapshot_test.go

```diff
@@ -0,0 +1,113 @@
+// Copyright © 2022 Meroxa, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package logrepl
+
+import (
+	"context"
+	"fmt"
+	"testing"
+	"time"
+
+	"github.com/conduitio/conduit-connector-postgres/test"
+	sdk "github.com/conduitio/conduit-connector-sdk"
+	"github.com/jackc/pgx/v4/pgxpool"
+
+	"github.com/matryer/is"
+)
+
+func TestLifecycle(t *testing.T) {
+	is := is.New(t)
+	ctx := context.Background()
+
+	pool := test.ConnectPool(ctx, t, test.RegularConnString)
+	table := createTestTable(t, pool)
```

Comment:
This can be simplified further, `test.SetupTestTable` already does everything we need:
```suggestion
	table := test.SetupTestTable(ctx, t, pool)
```